### ISSUE TITLE
绘制surface的问题

### ISSUE THREAD

**Issue by kickasss** [ecomfe/echarts-gl#319]
请问使用echarts4.5.0和echarts-gl1.1.2绘制surface时曲面的两端会自动连接起来是什么原因？我用的data是grid
![0001](https://user-images.githubusercontent.com/12770588/69480737-1ff67800-0e45-11ea-9670-bfed5485134d.JPG)
![0002](https://user-images.githubusercontent.com/12770588/69480754-2389ff00-0e45-11ea-9c45-140e7ae9b26b.JPG)

**Comment by oxydaemon** [comment]
I had similar issue with echarts 4.7, echarts-gl 1.1.2 

This appears to be caused by software thinking first and last data is a surface
I solved by sorting my Z data input tuples

Try sorting your Z data - I sorted [(X,Y,Z)] by X -> Y -> Z and showed up correctly

Also if you don't include any code / sample data in your issue its hard for them to troubleshoot :-)

**Comment by yunnuoyang** [comment]
> This appears to be caused by software thinking "first" and "last" data is a surface, so you see it draw a line between them
> I solved by sorting my Z data input tuples

yes,i also meet with this problem.after sort x,y,z,and x must appear twice more ,it can be handled.

**Comment by JeremKeith** [comment]
> > This appears to be caused by software thinking "first" and "last" data is a surface, so you see it draw a line between them
> > I solved by sorting my Z data input tuples
> 
> yes,i also meet with this problem.after sort x,y,z,and x must appear twice more ,it can be handled.

请问对z轴进行排序是什么意思，我现在是每组十个数据，x轴的值从小到大，y轴的值不变，z轴的值不定，这样排序的

**Comment by yunnuoyang** [comment]
意思就是x轴数据你从小到大假如是：1，2，3，4，这样绘制不出来的，不符合成图形的条件吧，你的数据应该是1，2，3，4，1这种，至少x有重复的数据。
